src/einsteinpy/plotting/geodesic/interactive.py

```diff
@@ -173,21 +186,25 @@ def plot(self, geodesic, color="#{:06x}".format(random.randint(0, 0xFFFFFF))):
         )
 
         _title_3D = dict(
-            text="Geodesic Plot", y=0.9, x=0.46, xanchor="center", yanchor="top"
+            text=title, y=0.9, x=0.46, xanchor="center", yanchor="top"
         )
 
         _scene_3D = dict(
             xaxis_title="X (GM/c^2)", yaxis_title="Y (GM/c^2)", zaxis_title="Z (GM/c^2)"
         )
 
-        self.fig.update_layout(title=_title_3D, scene=_scene_3D)
+        if aspect=="manual":
+            self.fig.update_layout(title=_title_3D, scene=_scene_3D, scene_aspectmode=aspect, scene_aspectratio=aspect_ratio)
+        else:
+            self.fig.update_layout(title=_title_3D, scene=_scene_3D, scene_aspectmode=aspect)
 
     def plot2D(
         self,
         geodesic,
         coordinates=(1, 2),
         figsize=(6, 6),
         color="#{:06x}".format(random.randint(0, 0xFFFFFF)),
+        title:str=None
```

Comment:
Use `title:str=''` since it follows the type annotation and the `if` check below is unaffected.